<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GURU - Financial audit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="reso.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Сохранить в</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScriptButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Обработать</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAuthor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gilmutdinov Ruslan - 2023
GitHub: RussellMorryson</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажите путь к файлу Excell для анализа или нажмите &quot;Открыть&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажите путь для сохранения результатов или нажмите &quot;Сохранить в&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Укажите путь к скрипту или нажмите &quot;Открыть&quot;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TextBoxOpen">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoPredictiveText</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TextBoxSave">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoPredictiveText</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TextBoxScript">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoPredictiveText</set>
    </property>
   </widget>
   <widget class="QLabel" name="Title_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Программа для анализа бухгалтерской отчетности юридических лиц</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="reso.qrc">
      <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="reso.qrc"/>
 </resources>
 <connections/>
</ui>
